ajax/libs/url-search-params/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "url-search-params",
+  "filename": "url-search-params.js",
+  "version": "0.7.0",
+  "license": "MIT",
+  "description": "Simple polyfill for URLSearchParams standard",
+  "keywords": [
+    "DOM",
+    "URLSearchParams",
+    "polyfill",
+    "shim"
+  ],
+  "author": {
+    "name": "Andrea Giammarchi",
+    "web": "https://webreflection.blogspot.com/"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/WebReflection/url-search-params.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/WebReflection/url-search-params.git",
+    "fileMap": [
+      {
+        "basePath": "build",
+        "files": [
+          "url-search-params.js"
```

Comment:
@pvnr0082t The author has replied. I think `url-search-params.js`(minified one) and `url-search-params.max.js`(not minified one) are what we need. Could you take a look at the author's reply to make sure if my understanding is correct. Here it is: WebReflection/url-search-params#18